app/posts/apply-for-teacher-training/2023-06-21-exploring-ways-to-integrate-the-services.md

```diff
@@ -0,0 +1,138 @@
+
+Anyone can [access their personal learner record](https://www.gov.uk/guidance/how-to-access-your-personal-learning-record), however this currently involves filling out and emailing a Word document along with proof of identity.
+
+An idea to explore in future might be to link these services together, allowing candidates to quickly import their qualifications into their teacher training application.
+
+![Screenshot showing a page asking for a unique learner number](school-qualifications.png)
+
+![Screenshot showing a page listing GCSE and A level qualifications imported from the learner record](import-qualifications.png)
+
+
+This would also have the advantage of verifying these qualifications, so that providers to not have to ask for proof of them later, which currently involves candidates finding (or re-requesting) their original certificates.
+
+## Messages
+
+There are several places in the service where candidates are encouraged to contact teacher training providers, with both email addresses and phone numbers given. Candidates can also contact support using an online chat feature (working hours only) or by phone. And if candidates have been assigned a teacher training adviser, they can get advice from them via email, phone, or SMS.
+
+These different communication channels may make it hard for candidates to keep track of. We also know that many candidates are reluctant to use phone communication.
+
+One idea to explore would be to have the option for candidates to send and receive messages within the service.
+
+![Screenshot showing a page listing messages from different providers](messages.png)
+
+![Screenshot showing a page with messages to and from a teacher tranining provider](tta-messages.png)
+
+This could supplement the other channels, with candidates also receiving new messages via email.
+
+The messages feature could also perhaps include automated notifications, such as when an offer is received from a provider.
+
+## Criminal records checks
+
+Once a candidate has accepted an offer for a training course, they are required to pass safeguarding checks including an enhanced criminal records check. This is currently all done off-service, with providers asked to manually mark the status of the check as ‘complete’ once it has been done.
+
+To make this more seamless, the service could give candidates a link to apply for a criminal records check, and then ask for their ‘share code’ once the check has been done.
+
+![Screenshot showing a page asking for a criminal records check share code](criminal-records-check.png)
+
+![Screenshot showing a page where criminal records checks have been successful](criminal-records-check-done.png)
+
+This would require tight integration with the [Disclosure and Barring Service (DBS)](https://www.gov.uk/government/organisations/disclosure-and-barring-service). Currently they don’t have an online service for enhanced checks, only basic ones, but this may change in future.
```

Comment:
```suggestion
This would require tight integration with the [Disclosure and Barring Service (DBS)](https://www.gov.uk/government/organisations/disclosure-and-barring-service). Currently they do not have an online service for enhanced checks, only basic ones, but this may change in future.
```